code/game/objects/items/weapons/cigs_lighters.dm

```diff
@@ -691,31 +693,38 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 			lit = TRUE
 			update_icon()
 			playsound(src.loc, pick(activation_sound), 75, 1)
-			if(istype(src, /obj/item/flame/lighter/zippo) )
-				user.visible_message(FONT_SMALL("Without even breaking stride, <b>[user]</b> flips open and lights \the [src] in one smooth movement."), range = 3)
-			else
+			if(istype(src, /obj/item/flame/lighter/zippo))
+				if(last_open <= world.time - 20) //Spam limiter.
+					last_open = world.time
+					user.visible_message(SPAN_NOTICE("Without even breaking stride, <b>[user]</b> flips open and lights \the [src] in one smooth movement."), range = 3)
```

Comment:
I can't say I like how you reverted these, the previous ones looked fine, but oh well

Reply:
The burn message felt kinda out of place, if you want a reason.